app/router.js

```diff
@@ -7,6 +7,8 @@ const Router = Ember.Router.extend({
 });
 
 Router.map(function() {
+  this.route('/');
```

Comment:
Why do you have this route in here? The root route will default to index.hbs / application.hbs